jar-service/pom.xml

```diff
@@ -200,8 +200,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<target>1.6</target>
-					<source>1.6</source>
+					<target>1.7</target>
+					<source>1.7</source>
```

Comment:
It should be updated to 1.7 everywhere or defined on the top level pom if possible.
Out of curiosity, what did you do which require 1.7 ?